MaxText/configs/models/mixtral-8x1b.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# model config for mixtral-8x1b
+
+base_emb_dim: 4096
+base_num_query_heads: 32
+base_num_kv_heads: 8
+base_mlp_dim: 14336
+base_num_decoder_layers: 5
```

Comment:
How did you choose these sizes? Generally we scale all "base" dimensions roughly proportionally, it looks like you only changed the layers here? Looks fine otherwise

Reply:
Yes, I only changed the number of layers to scale down to 8x1b.

Reply:
I suspect this doesn't converge as well as scaling each proportionally (would have to run experiments). Could we include a comment at the top of this file saying it is a custom scaling of mixtral-8x7b where we only change the layers for smaller scale performance testing, and we haven't run convergence experiments?

Reply:
Thanks Matt! After a second thought, I removed this small model config, and plan to add it into a branch.

Reply:
Sounds good, thank you!